src/pages/MySketches/index.jsx

```diff
@@ -1,20 +1,113 @@
+import { useState, useEffect } from "react";
+
 import NavBar from "../../components/NavBar";
 import SketchCard from "../../components/SketchCard";
+import Button from "../../components/Button";
 
 const MySketches = () => {
+  const [sketches, setSketches] = useState([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(null);
+
+  const onPrevButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage - 1);
+  };
+
+  const onNextButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage + 1);
+  };
+
+  const userId = sessionStorage.getItem("userEmail");
+
+  useEffect(() => {
+    const fetchUserSketches = async () => {
+      try {
+        if (!userId) {
+          console.error("No email found");
+
+          return;
+        }
+
+        const response = await fetch(
+          `http://localhost:3000/users/${userId}/sketches`,
+        );
+        const usersSkeches = await response.json();
+        setSketches(usersSkeches.sketches.list);
```

Comment:
```suggestion
        const usersSkeches = await response.json();
        
        setSketches(usersSkeches.sketches.list);
```

이 부분은 개행을 해주시는 것이 좋을 것 같습니다!

Reply:
수정했습니다~